packages/flutter/lib/src/painting/basic_types.dart

```diff
@@ -52,3 +52,41 @@ export 'dart:ui' show
 //  - window, WindowPadding
 //      These are generally wrapped by other APIs so we always refer to them directly
 //      as ui.* to avoid making them seem like high-level APIs.
+
+/// The description of the difference between two objects, in the context of how
+/// it will affect the rendering.
+///
+/// Used by [TextSpan.compareTo] and [TextStyle.compareTo].
+///
+/// The values in this enum are ordered such that they are in increasing order
+/// of cost. A value with index N implies all the values with index less than N.
+/// For example, [layout] (index 3) implies [paint] (2).
+enum RenderComparison {
+  /// The two objects are identical (meaning deeply equal, not necessarily
+  /// [identical]).
```

Comment:
This just hyperlinks right back to this enum value (as seen on https://master-docs-flutter-io.firebaseapp.com/flutter/painting/RenderComparison-class.html)